### ISSUE TITLE
Setting to disable dragging to open terminal

### ISSUE THREAD

**Issue by sguillia** [microsoft/vscode#112093]
#### TL;DR

Setting to opt-out of "drag to open terminal" because the right border of the window doesn't belong to the scrollbar anymore.

#### Description

This ticket is about a regression in the Insider build.

The regression was introduced by a feature called that I'll call "drag to open terminal".

On the Insider build, I can drag the utter right of the window to the left and it will open the workbench panel.

By workbench panel I mean the panel containing the terminal, debug console etc.

This feature is referred to in #111463

This feature seems to be introduced in Insider and seems to be absent in Code 1.51.1

#### Problem

When I want to scroll, I can just move my mouse completely to the right without caring.  
At that point, my cursor has the default pointer.
When I click, the editor will scroll to the desired position.
This is great.

Now on the Insider build:
When I want to scroll, I move my mouse completely to the right without caring.
At that point, my cursor has the *drag* appearance.
When I click, nothing happens (because window expected me to drag)
This is not great.

I'd like a setting to opt-out of this feature, or any other experience where I do not have to precisely focus my mouse pointer on the scroll bar instead of the right border of the window that used to belong to the scroll bar.

It's like the "close" button on macOS windows, you have to precisely click on that ridiculously small button instead of just pushing your mouse top right.

#### Version

```
Version: 1.51.1
Commit: e5a624b788d92b8d34d1392e4c4d9789406efe8f
Date: 2020-11-10T23:31:29.624Z
Electron: 9.3.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Linux x64 4.15.0-123-generic

Version: 1.52.0-insider
Commit: 92192baf760315ac47fb2caaf3ce6d0778962352
Date: 2020-12-03T05:44:34.000Z
Electron: 9.3.5
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Linux x64 4.15.0-123-generic
```

**Comment by sguillia** [comment]
Does not reproduce in 1.52.1  
Does not reproduce in 1.53.0-insider

I guess the regression never made its way to the stable build

Thank you

**Comment by sguillia** [comment]
This problem happens again, but only in full-screen mode.

```
Version: 1.53.0-insider
Commit: 861a88ebadb4335df92251faf6420b598440e082
Date: 2021-01-29T21:49:47.914Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Linux x64 4.15.0-133-generic
```

**Comment by isra-fel** [comment]
+1 for horizontal scroll bar as well in zen mode
Please make the "drag to open panel" feature configurable. Thanks

**Comment by Tyriar** [comment]
Am I understanding this right that it's about the red line (right most pixel) below not scrolling the terminal?

![image](https://user-images.githubusercontent.com/2193314/119509638-ab481680-bd25-11eb-8665-847d6034dedd.png)

**Comment by isra-fel** [comment]
Hey @Tyriar , I believe @sguillia 's probem was about the same red line not scrolling the **editor** in full-screen mode, but opens the panel (terminal) instead.
I also believe they set the default location of the panel to **right**.

**Comment by Tyriar** [comment]
Oh OK, thanks for the clarification

**Comment by sguillia** [comment]
@sbatten This bug is fixed in 1.63.2

If it was as designed, then your design has changed and is now better. I hope it will stay
